airbyte-webapp/src/pages/SourcesPage/pages/CreateSourcePage/CreateSourcePage.tsx

```diff
@@ -35,11 +38,22 @@ const CreateSourcePage: React.FC = () => {
 
   return (
     <>
-      <HeadTitle titles={[{ id: "sources.newSourceTitle" }]} />
-      <PageTitle withLine title={<FormattedMessage id="sources.newSourceTitle" />} />
-      <FormPageContent>
-        <SourceForm onSubmit={onSubmitSourceStep} sourceDefinitions={sourceDefinitions} hasSuccess={successRequest} />
-      </FormPageContent>
+      <HeadTitle titles={[{ id: "sources.newSourceTitle" }]} />{" "}
+      <DocumentationPanelProvider>
+        <ConnectorDocumentationLayout>
```

Comment:
These could be combined into a single "container" component since the two are always used together. That way the consuming pages just need to set up a single component.